03-objects-and-composition/exercise/07.js

```diff
@@ -0,0 +1,25 @@
+function townsToJSON(array){
+  const [, ...items] = array
+  //  regexp to match all spaces and |
+  let regex = /\s?\|\s?/;
```

Comment:
When we are not planning to change a variable value it's always better to declare it with `const`.